mount/mountinfo.go

```diff
@@ -54,3 +56,27 @@ type Info struct {
 	// VFSOptions represents per super block options.
 	VFSOptions string
 }
+
+func fromMountinfoSlice(infos []*mountinfo.Info) []Info {
+	out := make([]Info, len(infos))
+	for i, v := range infos {
+		out[i] = fromMountinfo(v)
+	}
+	return out
+}
+
+func fromMountinfo(info *mountinfo.Info) Info {
+	return Info{
+		ID:         info.ID,
+		Parent:     info.Parent,
+		Major:      info.Major,
+		Minor:      info.Minor,
+		Root:       info.Root,
+		Mountpoint: info.Mountpoint,
+		Options:    info.Options,
+		Optional:   info.Optional,
+		FSType:     info.FSType,
+		Source:     info.Source,
+		VFSOptions: info.VFSOptions,
```

Comment:
OK, now the fields are the same, so why don't we do something like

```go
type Info = mountinfo.Info
```

and drop the `fromMountinfo` entirely?

wdyt @thaJeztah?

Reply:
Ahh, my bad, I see you're doing it in the next commit. Not sure if this intermediate step makes sense (iow I think this commit and the next one may need to be merged)

Reply:
Ah, good one; I think it was separate because I had a temporary vendor in between, but yes, I can see it now doesn't make sense to implement (and remove) the conversion code in a single PR; I can squash them.

Reply:
squashed the commits